components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -75,7 +89,9 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 			'en': {
```

Comment:
Maybe not for this PR but should these translations get pulled into an external file? I can imagine this file getting quite long once they are all filled in.

Reply:
I agree that it should get pulled into an external file. `d2l-filter-dropdown-category.js` is another example of a file that does this and it's localization code should also probably move to an external file.

There are some places in this repo where it has been extracted to another file, example: `d2l-filter-dropdown-localize-behavior.js`. I feel like the best time to move it to an external file in when we start to fill it up with more translations.

Reply:
Definitely not for this PR, since this PR doesn't touch langterms. We can file a separate issue/story/whatever for that. (And more importantly IMO, getting this set up for automated translations)

Reply:
Definitely not for this PR, since that's increasing the scope. But more important than the file size to me is that this repo isn't part of our automated translations process, so separately, we should either do that (which almost certainly requires splitting out these files regardless of size), or requesting translations directly